pkg/configs/db/db.go

```diff
@@ -24,12 +24,12 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 
 // RulesDB has ruler-specific DB interfaces.
 type RulesDB interface {
-	// GetRulesConfig gets the user's alertmanager config
+	// GetRulesConfig gets the user's ruler config
```

Comment:
Please include this change in the commit description, or move to a separate commit if that makes more sense.

Reply:
I've moved it to be part of 029f17c, which introduces this interface.